upmc-enterprises/cda2fhir
# Upmcfhir 277

#### What does this PR do?

Implements bundle 'ifNoneExists' logic to prevent duplicates from being created on the server. The identifier logic may be optionally over-ridden in the configuration file if a specific OID/URL is to be used.

#### Related JIRA tickets:

UPMCFHIR-277

#### How should this be manually tested?

Run the integration tests, and make sure they pass. You may only run them one at a time, as running them together will impact the counts of the others as they aren't boxed by patient identifier. Only the Epic and the test CCD counts currently test that the logic works.

#### Background/Context:

Be aware the medication deduplication doesn't rely on the same organization deduplication logic by manufacturer that steven implemented. However, none of the organizations underlying a medication have an identifier on the manufacturer on any of the Epic samples I've seen (or any of the samples for that matter), so I'm not sure it matters. Welcome to feedback on that though.

#### New JIRA tickets for clinical review required?
